modules/rostests/apitests/kernel32/ConsoleCP.c

```diff
@@ -355,7 +355,7 @@ static void test_cp932(HANDLE hConOut)
         c.Y = 0;
         ret = ReadConsoleOutputCharacterW(hConOut, str, 3 * sizeof(WCHAR), c, &len);
         ok(ret, "ReadConsoleOutputCharacterW failed\n");
-        ok(len == 4, "len was: %ld\n", len);
+        ok(len == 4, "len was: %ld\n", len);                                    // BUG BUG ??
```

Comment:
bug because?
Please elaborate in a comment, so the next one reading it does not have to investigate himself.

Reply:
That's a local investigation by myself that I put in the branch to have it available.
According to the references I linked above in the "description" of the draft PR what the ReadConsoleOutputCharacter() function can return heavily depends on:
- the current code page of the console,
- AND, the current font being used. Indeed even if the console is using CJK code pages, it is not yet guaranteed that CJK characters that we would naively think of as being full-width (thus taking two cells) really take that much space, because it is also related to how they appear with the corresponding selected font. So there are still cases where if the font makes all the characters take one cell (incl. the full-width ones) then the latter are not reported as full-width, and the returned length from ReadConsoleOutputCharacter() will be different (as well as the corresponding character attributes).

I linked also the test results of this apitest that I sent to the Wine testbots (with one chinese and one japanese windows VM) and the test fail at some places.

Also the BUG BUG was to warn me that I have to check again ReadConsoleOutputCharacter() implementation because when I tested the test on ROS the returned length was different than 4.